scripts/cibuild/SetVstsBuildNumber.ps1

```diff
@@ -0,0 +1,41 @@
+# This script is used to dynamically set some build variables during VSTS build.
+# Specifically, this script reads the buildcounter.txt file in the $(DropRoot) to
+# determine the build number of the artifacts, also it sets the $(NupkgOutputDir)
+# based on whether $(BuildRTM) is true or false.
+param
+(
+    [string]$BuildCounter,
+    [string]$BuildInfoJson,
+    [string]$BuildRTM
```

Comment:
@alpaix VSTS limitation for parallel builds, it only passes the string value out of [true,false]